<launch>
    <node name="stage" pkg="stage_ros" type="stageros"
	    args="$(find lab4)/world/playground.world" />

    <param name="goal_x" type="double" value="4.5" />
    <param name="goal_y" type="double" value="9.0" />

    <node name="star_trek" pkg="lab4" type="star_trek.py"/>

</launch>